pkgs/top-level/aliases.nix

```diff
@@ -879,4 +879,11 @@ mapAliases ({
     sddm
   ;
 
+  /* https://github.com/NixOS/nixpkgs/issues/107280 */
+  deluge-1_x = lib.trivial.warn ''
+    Deluge 1.x will be removed in NixOS 21.09. Please back up your configuration
```

Comment:
```suggestion
    Deluge 1.x will be removed in NixOS 21.11. Please back up your configuration
```

Reply:
@jonringer: is the current nixpkgs master version on purpose: `lib.version`?
"21.03.git.3a3da0950d5" or do we do the switch in 21.11 and not in 21.03?

Reply:
@Mic92: The next one is 21.05, and the one after that 21.11. See this PR here: https://github.com/NixOS/nixpkgs/pull/108899